microsetta_admin/templates/scan.html

```diff
@@ -305,6 +311,19 @@ <h4>Scan History</h4>
                 </td>
             </tr>
             <!-- End collapsible email template fields -->
+            {% if observations %}
+                <tr>
+                    <td>Observations:</td>
+                    <td>
+                        {% for observation in observations %}
```

Comment:
The most recent change lost the concept of grouping into categories:

![observation_grouping](https://github.com/biocore/microsetta-admin/assets/83246693/5d266fae-990f-4833-b05c-2f2bc55c5320)